Libraries/Utilities/HMRClient.js

```diff
@@ -73,6 +73,7 @@ Error: ${e.message}`
     };
     activeWS.onmessage = ({data}) => {
       // Moving to top gives errors due to NativeModules not being initialized
+      const { AndroidConstants } = require('NativeModules');
       const HMRLoadingView = require('HMRLoadingView');
```

Comment:
you can use the `host` and `port` param `HMRClient` receives on the constructor for iOS